src/functions/getTestResultsBySystemNumber.ts

```diff
@@ -70,7 +70,7 @@ export const getTestResultsBySystemNumber = async (event: {
                 }
             }
         }
-        return testResultsService.getTestResults({systemNumber, testStatus, fromDateTime, toDateTime, testResultId, testVersion})
+        return testResultsService.getTestResultBySystemNumber({systemNumber, testStatus, fromDateTime, toDateTime, testResultId, testVersion})
```

Comment:
Can we flatten these out into await calls. we can then simplify the logic in the called function

Reply:
will do in 18734